### PR TITLE
Remove Obsolete MINDS-i Repos

### PR DESCRIPTION
**Pull request by agyoungs**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @agyoungs . I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/11/12 15:17:59 Loaded 7459 libraries from DB
Removing MINDSi
Success!
```
```
2024/11/12 15:18:46 Loaded 7458 libraries from DB
Removing MINDS-i-Trailer
Success!
```
```
2024/11/12 15:19:36 Loaded 7457 libraries from DB
Removing MINDS-i-Utils
Success!
```

**Pull request merged**
